crates/iota-graphql-client/src/query_types/epoch.rs

```diff
@@ -129,5 +127,5 @@ pub struct ValidatorSet {
     /// Size of the validator candidates `Table`.
     pub validator_candidates_size: Option<i32>,
     /// Object ID of the `Table` storing the validator candidates.
-    pub validator_candidates_id: Option<Address>,
+    pub validator_candidates_id: Option<ObjectId>,
```

Comment:
I still feel like address was better for things like this. @Thoralf-M @Alex6323 opinions?

Reply:
https://github.com/iotaledger/iota/issues/8133

Reply:
Thanks